apps/daimo-mobile/src/storage/storedTypes.ts

```diff
@@ -233,7 +233,6 @@ export interface StoredV16ProposedSwap {
 interface StoredV16ForeignCoin {
   chainId: number;
   token: Address;
-  name: string;
```

Comment:
just checking this won't break backcompat right

Reply:
yeah great question. i thought about it & i think it's fine in this case.

in general you're absolutely right, we should not change existing Stored... models